securedrop_client/logic.py

```diff
@@ -266,7 +267,8 @@ def on_login_timeout(self):
         """
 
         self.api = None
-        error = _('The connection to SecureDrop timed out. Please try again.')
+        error = _('The connection to the SecureDrop server timed out. '
```

Comment:
+1 for saying "SecureDrop sever" because to a journalist, the thing they're using *is* SecureDrop.